eng/targets/Helix.Common.props

```diff
@@ -37,7 +37,7 @@
       <!-- aspnetcore-quarantined-tests (quarantined-tests.yml) and RunHelix.ps1 -RunQuarantinedTests -->
       <ItemGroup>
         <!-- Linux -->
-        <HelixAvailableTargetQueue Include="Redhat.7.Amd64.Open" Platform="Linux" />
+        <HelixAvailableTargetQueue Include="(AlmaLinux.8.Amd64.Open)Ubuntu.1804.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:almalinux-8-helix-amd64" Platform="Linux" />
```

Comment:
Use a variable like https://github.com/dotnet/aspnetcore/pull/47970/files#diff-fecf6f98b5c2a5cc5c66f045a93d30421b7fedcbdd4cacefc2605cd61a9b89afL8? Then you don't need to copy this string across files